src/plopHelpers.js

```diff
@@ -28,7 +28,7 @@ async function getGenerator() {
 							placeHolder: "Please choose a generator"
 						}
 				  );
-		return plop.getGenerator(generator.label) || plop.getGenerator(generator.name);

```

Comment:
This was causing an error because when `generator.label === undefined`, `plop.getGenerator(undefined)` throws an error